src/instanceMethods/close.js

```diff
@@ -70,23 +70,11 @@ export function close (resolveValue) {
 
   const innerParams = privateProps.innerParams.get(this)
   const swalPromiseResolve = privateMethods.swalPromiseResolve.get(this)
-  const onClose = innerParams.onClose
-  const onAfterClose = innerParams.onAfterClose
+  const { onClose } = innerParams
 
-  dom.removeClass(popup, swalClasses.show)
-  dom.addClass(popup, swalClasses.hide)
+  replacePopupClass({ popup })
```

Comment:
The choice of the name for the function isn't the best, do we need this function at all? I think codeclimate will be happy even without introducing `replacePopupClass()`

Reply:
really did not need to. It worked fine without the function.